nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/test/java/org/nuxeo/ecm/automation/core/operations/blob/BlobToFileTest.java

```diff
@@ -134,10 +149,21 @@ protected OperationContext buildCtx(CoreSession coreSession) throws IOException
         return ctx;
     }
 
-    protected Map<String, Object> buildParams() {
+    protected Map<String, Object> buildParams(File directory) {
         Map<String, Object> params = new HashMap<>();
-        params.put("directory", targetDirectory.getAbsolutePath());
+        params.put("directory", directory.getAbsolutePath());
         params.put("prefix", "");
         return params;
     }
+
+    protected File createAllowedTargetDirectory() throws IOException {
+        File directory = File.createTempFile("target", "", new File("/tmp"));
```

Comment:
`Framework.createTempDirectory` creates a temp directory in `/nuxeo/nuxeo-features/nuxeo-automation/nuxeo-automation-core/target`
And in the unit test, nuxeoHome directory is also in `/nuxeo/nuxeo-features/nuxeo-automation/nuxeo-automation-core/target`